modules/openapi-generator/src/main/resources/python/rest.mustache

```diff
@@ -4,34 +4,36 @@
 
 import io
 import json
-import logging
 import re
 import ssl
 
 from urllib.parse import urlencode, quote_plus
 import urllib3
 
-from {{packageName}}.exceptions import ApiException, UnauthorizedException, ForbiddenException, NotFoundException, ServiceException, ApiValueError, BadRequestException
-
-
-logger = logging.getLogger(__name__)
+from {{packageName}}.exceptions import ApiException, ApiValueError
 
+RESTResponseType = urllib3.HTTPResponse
 
 class RESTResponse(io.IOBase):
 
     def __init__(self, resp) -> None:
-        self.urllib3_response = resp
```

Comment:
`RESTResponse.urllib3_response`, `RESTResponse.tornado_response`, and `RESTResponse.aiohttp_response` have been changed to response.
This is necessary to make them easier to handle in `without_preload_content`.